libraries/AP_Notify/Buzzer.cpp

```diff
@@ -204,6 +104,25 @@ void Buzzer::update()
     }
 }
 
+
+void Buzzer::update_playing_pattern()
+{
+    if (_pattern == 0UL) {
+        return;
+    }
+
+    const uint32_t now = AP_HAL::millis();
+    const uint32_t delta = now - _pattern_start_time;
+    if (delta > 3100) {
```

Comment:
This means the last bit might actually not be played, as it will only be played if delta is exactly 3100.

I think this should be:
```suggestion
    if (delta >= 3200) {
```

Reply:
Fixed, thanks!